lua/lv-which-key/init.lua

```diff
@@ -198,27 +178,26 @@ if O.extras then
         t = {"<cmd>VimtexTocToggle<cr>", "Toggle Table Of Content"},
         v = {"<cmd>VimtexView<cr>", "View PDF"}
     }
+    normalMappings["p"] = {"<cmd>lua require'telescope'.extensions.project.project{}<cr>", "Projects"}
+    normalMappings["r"] = {
+        name = "Replace",
+        f = {"<cmd>lua require('spectre').open_file_search()<cr>", "Current File"},
+        p = {"<cmd>lua require('spectre').open()<cr>", "Project"}
+    }
+    normalMappings["z"] = {
+        name = "Zen",
+        s = {"<cmd>TZBottom<cr>", "toggle status line"},
+        t = {"<cmd>TZTop<cr>", "toggle tab bar"},
+        z = {"<cmd>TZAtaraxis<cr>", "toggle zen"}
+    }
+    normalMappings[";"] = {"<cmd>Dashboard<cr>", "Dashboard"}
+    visualMappings["r"] = {
+        name = "Replace",
+        f = {"<cmd>lua require('spectre').open_visual({path = vim.fn.expand('%')})<cr>", "File"},
```

Comment:
Spectre will have the same problem as CommentToggle, replace \<cmd> with : so that it catches the selection.